bot.py

```diff
@@ -527,13 +545,19 @@ async def play(self, message, command_content):
             logging.info("Playing media")
             self.next_in_queue()
 
+    async def play_next(self, message, command_content):
+        """
+        Like play, but puts the song at the front of the queue.
+        """
+        await self.play(message, command_content, deque_append_left=True)
```

Comment:
deque_append_left doesn't match play_next in the play function.

Reply:
good catch :)